utils/gyb_syntax_support/DeclNodes.py

```diff
@@ -96,11 +96,13 @@
     # if-config-decl -> '#if' expr stmt-list else-if-directive-clause-list
     #   else-clause? '#endif'
     Node('IfConfigDecl', kind='Decl',
-         traits=['WithStatements'],
          children=[
              Child('PoundIf', kind='PoundIfToken'),
              Child('Condition', kind='Expr'),
-             Child('Statements', kind='CodeBlockItemList'),
+             Child('Elements', kind='Syntax',
+                   node_choices=[
+                      Child('Statements', kind='CodeBlockItemList'),
+                      Child('SwitchCases', kind='SwitchCaseList')]),
```

Comment:
Dropping `WithStatements` trait from `IfConfigDecl`.
But separating `IfConfigDecl` only for switch-cases (i.e. `SwitchCaseIfConfig`) doesn't seem the right choice.